portfolio/src/main/webapp/script.js

```diff
@@ -26,17 +26,37 @@ document.querySelector('.comment-form').addEventListener('submit', async functio
     console.log('Could not submit comment');
     return;
   }
-  appendComment(document.querySelector('.comment-box').value);
+  rerequestComments();
 });
 
+// Load the dynamic parts of the page like comments and the authentication status.
+async function load() {
+  const response = await fetch(`/comment?num=5`);
+  const json = await response.json();
+  appendComments(json.comments);
```

Comment:
Call getComments(5) instead?

Reply:
I use the `json` variable below this to get the user's email.